avocado/utils/memory.py

```diff
@@ -41,6 +108,20 @@ def read_from_meminfo(key):
     return int(re.search(r'\d+', meminfo).group(0))
 
 
+def swaptotal():
+    """
+    Read ``SwapTotal`` from meminfo.
+    """
+    return read_from_meminfo('SwapTotal')
+
+
+def freeswap():
+    """
+    Read ``SwapFree`` from meminfo.
+    """
+    return read_from_meminfo('SwapFree')
```

Comment:
Ditto. Also, this should not be in this commit. But I'd not propose these 2 functions anyway.